src/Android/Xamarin.Android/Samples/Symbology/FeatureLayerExtrusion/FeatureLayerExtrusion.cs

```diff
@@ -125,7 +125,7 @@ private void ChangeExtrusionExpression()
             // Toggle the feature layer's scene properties renderer extrusion expression and change the button text
             if (_button_ToggleExtrusionData.Text == "Population Density")
             {
-                myRendererSceneProperties.ExtrusionExpression = "[POP07_SQMI] * 5000";
+                myRendererSceneProperties.ExtrusionExpression = "[POP07_SQMI] * 5000 + 100000";
```

Comment:
Should we describe this expression (explain why we're manipulating the raw pop density, e.g.)?
Maybe something like:
`// Normalize the population density values for displaying extruded features`

 It's probably OK as is, since it's not directly relevant to what the sample is illustrating. I'll leave it up to you.

Reply:
That's a good point, that concept is critical to explain to the user.